apps/site/pages/en/eol.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: End-Of-Life
+layout: article
+modal: eol
+---
+
+# End-Of-Life (EOL)
+
+## Why and how Node.js releases reach End-Of-Life
+
+Major versions of Node.js are released, patched, and designated End-Of-Life on a predictable schedule. As it's not feasible to maintain all release lines in perpetuity, after a planned maintenance period, a Node.js major release line will stop being maintained by the project.
+
+[View the Node.js release schedule](/about/releases/).
+
+<div className="flex flex-col items-start gap-4 xl:flex-row xl:items-center">
+  <Button kind="primary" href="/download" className="flex-1">
+    <span>Upgrade to the latest LTS Node.js®</span>
```

Comment:
I don't know if that works in MDX, we might be able to get rid of the `<span>` elements using `Fragment` syntax
```suggestion
    <>Upgrade to the latest LTS Node.js®</>
```

Reply:
What prevents us from putting the text as a direct child of the button?

Reply:
See https://github.com/nodejs/nodejs.org/pull/7990/commits/5ab2cbb002dbd7459d89f0192c20096527e579be, TL;DR an additional `<p>` is added which breaks a11y